web_widget_prefixed_url/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 - TODAY Serpent Consulting Services Pvt. Ltd.
+#                            (<http://www.serpentcs.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Web Widget Prefixed URL",
+    'version': "11.0.1.0.0",
+    'author': "Serpent Consulting Services Pvt. Ltd., "
+              "Odoo Community Association (OCA)",
+    'category': 'Web',
+    'website': "https://github.com/OCA/Web",
```

Comment:
Change "https://github.com/OCA/Web" to  "https://github.com/OCA/web" (lower case)